src/main/java/com/microsoft/azure/functions/binding/RpcDataTargets.java

```diff
@@ -24,23 +24,27 @@
     @Override
     public void setStatus(int status) { this.status = status; }
     @Override
-    public void addHeader(String key, String value) { this.headers.put(key, value); }
+    public void addHeader(String key, Object value) { this.headers.put(key, value); }
```

Comment:
Make sure you updated the signature of [the interface in java-library](https://github.com/Azure/azure-functions-java-library/blob/dev/src/main/java/com/microsoft/azure/functions/HttpResponseMessage.java#L6-L7).